tests/test_streams.py

```diff
@@ -377,23 +377,24 @@ async def test_readline_exception(self) -> None:
         with pytest.raises(ValueError):
             await stream.readline()
 
-    async def test_readuntil(self) -> None:
+    @pytest.mark.parametrize("separator", [b"*", b"**"])
+    async def test_readuntil(self, separator: bytes) -> None:
         loop = asyncio.get_event_loop()
         # Read one chunk. 'readuntil' will need to wait for the data
         # to come from 'cb'
         stream = self._make_one()
         stream.feed_data(b"chunk1 ")
-        read_task = loop.create_task(stream.readuntil(b"*"))
+        read_task = loop.create_task(stream.readuntil(separator))
 
         def cb():
             stream.feed_data(b"chunk2 ")
-            stream.feed_data(b"chunk3 ")
-            stream.feed_data(b"* chunk4")
+            stream.feed_data(b"chunk3 " + separator)
+            stream.feed_data(b" chunk4")
```

Comment:
This one seems to have changed. Deliberate, or should we prefix with `separator`?

Reply:
I'll do it like before
What do you think about the other tests?
Should I use pytest's parametrization decorator in other tests?

Reply:
Would probably be nice, just to have that extra coverage, but it's up to you.